src/main/java/uk/gov/dvsa/model/cvs/certificateData/IvaFailCertificateData.java

```diff
@@ -145,4 +137,12 @@ public RequiredStandard[] getRequiredStandards() {
     public void setRequiredStandards(RequiredStandard[] requiredStandards) {
         this.requiredStandards = requiredStandards;
     }
+
```

Comment:
Did this class need to be modified as the reapplicationDate was already there?

Reply:
I'll revert it now